lib/src/container/store.rs

```diff
@@ -487,3 +487,13 @@ pub fn prune_images(_repo: &ostree::Repo, _imgs: &[&str]) -> Result<()> {
     // And remove the unreachable ones.
     unimplemented!()
 }
+
+impl From<cli::ContainerProxyOpts> for ImageProxyConfig {
```

Comment:
While this is just one crate today, this feels like a layering violation to have the `lib/` code import `cli/`.

IOW I feel the clippy lint is correct in general, but in this specific case using `Into` is better, and we can just annotate with `#[allow(clippy::from_over_into)]`.  
Do you agree?

Reply:
No strong feelings on either approaches, I can push it in that direction.

But just to double-check, are you perhaps misquoting this? Both the source and target types here are in the same `lib/` crate, we never cross to the `cli/` one.

Reply:
Right, today the CLI code is actually all in `lib/` so that rpm-ostree can re-export it.

But the principle still stands I think.  

(Also we could fix this by keeping the `From<>` implementation in `cli.rs` right?)

Reply:
Ack. Then I'll happily revert my change and just silence clippy on this one. The previous code arrangement is fine for me.